picker/index.html

```diff
@@ -8,29 +8,15 @@
 	<script src="index.js" type="module"></script>
 	<link rel="icon" id="favicon">
 	<script src="https://stretchy.verou.me/dist/stretchy.iife.min.js" data-stretchy-filter="autosize" async defer></script>
+	<script src="https://elements.colorjs.io/src/color-picker/color-picker.js" type="module"></script>
```

Comment:
Nit: I would prefer an `import` in the main JS file (the other one is not ESM)

Reply:
Good point. Will fix it. Thanks!